### PR TITLE
Remove reference to Basic Authentication in the docs

### PR DESCRIPTION
**Pull request by rkreisel**
Edited the section on Authentication to remove Basic Auth, which GitHub no longer supports, with instructions for using a Personal Access Token.

### CODE REVIEW

**Comment by JonruAlveus** [comment]
Fixes #2554

**Review by JonruAlveus — APPROVED** [review]
Thanks for the contribution! ⭐

**Comment by rkreisel** [comment]
Hey Chris,

I've only ever used Git Hub for my personal projects. This was the first PR I have ever done to a public project. Is there something further I need to do to get this moved to the main branch?

Thanks,
Randy

----------------------------------------

File: docs/getting-started.md [hunk 1]
```diff
@@ -16,15 +16,15 @@ The API will reject you if you don't provide a `User-Agent` header (more details
...
+There are two Authentication options supported by the API. 
 
-```csharp
-var basicAuth = new Credentials("username", "password"); // NOTE: not real credentials
-client.Credentials = basicAuth;
-```
+- Personal Access Tokens (PAT) and 
+- OAuth authentication.
+
+If you wish to use a PAT follow this [link](https://docs.github.com/en/authentication/keeping-your-account-and-data-secure/creating-a-personal-access-token) to generate one, then replace "token" in the snippet below with the text generated by GitHub.
```
Comment (JonruAlveus) [inline docs/getting-started.md:24]: Please could you remove the /en from the url? That way the GitHub website can work out what language it should display for the user 😄

**Review by JonruAlveus — CHANGES_REQUESTED** [review]
Sorry just noticed one thing

**Comment by JonruAlveus** [comment]
> Hey Chris, I've only ever used Git Hub for my personal projects. This was the first PR I have ever done to a public project. Is there something further I need to do to get this moved to the main branch? Thanks, Randy
> […](#)
> ---------------------------------------- From: "Chris Simpson" ***@***.***> Sent: 8/25/22 11:42 PM To: "octokit/octokit.net" ***@***.***> Cc: Randy Kreisel ***@***.***>, Author ***@***.***> Subject: Re: [octokit/octokit.net] Update getting-started.md (PR #2555) @JonruAlveus approved this pull request. Thanks for the contribution! ⭐ — Reply to this email directly, view it on GitHub, or unsubscribe. You are receiving this because you authored the thread.

Hi Randy,

Firstly, thanks for the contribution! Around merging the PR, after the one change I think we're good to go. Once that's done I or one of the other Contributers can merge the PR. I believe our primary merger is currently on holiday 😄

File: docs/getting-started.md (hunk 1 above)
Comment (rkreisel) [reply docs/getting-started.md]: Found two or three instances and removed them.

File: docs/getting-started.md [hunk 2]
```diff
@@ -3,28 +3,28 @@
 The easiest way to get started with Octokit is to use a plain `GitHubClient`:
 
 ```csharp
-var client = new GitHubClient(new ProductHeaderValue("my-cool-app"));
+var client = new GitHubClient(new ProductHeaderValue("my-cool-app"));removed
```
Comment (timrogers) [inline docs/getting-started.md:6]:
```suggestion
var client = new GitHubClient(new ProductHeaderValue("my-cool-app"));
```

Did you mean to add "removed" here?

File: docs/getting-started.md [hunk 3]
```diff
@@ -3,28 +3,28 @@
...
 
-The API will reject you if you don't provide a `User-Agent` header (more details [here](https://docs.github.com/en/rest/overview/resources-in-the-rest-api#user-agent-required)). This is also to identify applications that are accessing the API and enable GitHub to contact the application author if there are problems. So pick a name that stands out!
+The API will reject you if you don't provide a `User-Agent` header (more details [here](https://docs.github.com/rest/overview/resources-in-the-rest-api#user-agent-required)). This is also to identify applications that are accessing the API and enable GitHub to contact the application author if there are problems. So pick a name that stands out!
 
 ### Authenticated Access
 
 If you want to access private repositories or perform actions on behalf of a user, you need to pass credentials to the client.
 
-There are two options supported by the API - basic and OAuth authentication.
+There are two Authentication options supported by the API.
```
Comment (timrogers) [inline docs/getting-started.md:19]: This isn't quite true. But that's not your fault, as GitHub's authentication options are very very complicated 🙈  I think I would correct this and just say something like this:

```suggestion
You can authenticate by passing an access token, for example a personal access token (PAT), an OAuth access token or an installation token from a GitHub App
```

Comment (timrogers) [reply docs/getting-started.md:19]: (As well as accepting this suggestion, you would need to remove the bullet pointed list below it.)

**Review by timrogers — CHANGES_REQUESTED** [review]
Thanks for taking the time to make this improvement to our docs ❤️ I'd like to make some suggestions to make it even better.

**Review by timrogers — APPROVED** [review]

**Comment by nickfloyd** [comment]
release_notes: Removed reference to Basic Authentication in the docs

**Pull request merged**